Source/WebCore/Modules/credentialmanagement/DigitalCredential.idl

```diff
@@ -26,7 +26,8 @@
 [
     Exposed=Window,
     SecureContext,
-    EnabledBySetting=DigitalCredentialsEnabled
-] interface DigitalCredential {
+    EnabledBySetting=DigitalCredentialsEnabled,
+    Conditional=WEB_AUTHN,
+] interface DigitalCredential : BasicCredential {
```

Comment:
This change is web-facing so we should have a test for it.

Reply:
I added just a simple existence test, as I figure we will start adding actual WPTests once the spec starts getting padded out.